pennylane/transforms/batch_params.py

```diff
@@ -71,29 +71,29 @@ def circuit(x, weights):
 
     >>> batch_size = 3
     >>> x = np.linspace(0.1, 0.5, batch_size)
-    >>> weights = np.random.random((batch_size, 10, 3, 3))
+    >>> weights = np.random.random((batch_size, 10, 3, 3), requires_grad=True)

```

Comment:
```suggestion
    >>> rng = np.random.default_rng(seed=1234)
    >>> weights = rng.random((batch_size, 10, 3, 3), requires_grad=True)
```

A change I was wanting to make to this page anyway.  This will make the output deterministic and users can compare what they get to printed output.

Reply:
I think it's reasonable, but frankly not exactly mandatory. We're showcasing that the feature works well, rather than showing exact numerical data that users should check themselves. Having said that, it might lead to confusion, if they do not see the same results, so happy to accept these suggestions.